clustershell.spec.in

```diff
@@ -21,7 +21,7 @@
 %{!?__python3: %global __python3 python3}
 %{!?python3_shortver: %global python3_shortver %(%{__python3} -c 'import sys; print(str(sys.version_info.major) + "." + str(sys.version_info.minor))')}
 
-%if ! 0%{?rhel} >= 8 && ! 0%{?suse_version} > 1550
+%if 0%{?rhel} < 8 &&  0%{?suse_version} <= 1500
```

Comment:
Matt in the issue only inverted the equality, but you also changed the suse version from 1550 to 1500 -- is that on purpose?
(might want to say that in the commit message if so)

Reply:
It was unintentional - I don't have OpenSuSE so I'm not sure exactly what version stopped supporting py2. Changing the 1500 back to 1550 is probably the right thing to do.

Reply:
%suse_version 1500 includes Leap/SLE 15 - which still has python2. %suse_version 1550 is Tumbleweed/Factory (currently) where python2 has been dropped from.

Reply:
Ok, so from that I take we don't want python2 in Tumbleweed (1550), so the new logic is correct -- wasn't that just what the problem is about?

Well, non-convoluted logic is better for everyone anyway, thanks!